monitor/nuro-data-collector/src/main/java/config/Environment.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2015 SeaCloudsEU
+ * Contact: Michele Guerriero <michele.guerriero@mail.polimi.it>
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package config;
+
+public class Environment {
```

Comment:
It was just to avoid to have strings within the code in the case in the future environmental variables would change (actually during twoer 4cloud developmente they have been changed several times even if now they should be stable). Anyway I can remove this mapping, do you suggest to hard code the variable names?